indexer/api/handlers/deposits.go

```diff
@@ -29,7 +29,7 @@ type DepositResponse struct {
 
 // TODO this is original spec but maybe include the l2 block info too for the relayed tx
 func newDepositResponse(deposits []*database.L1BridgeDepositWithTransactionHashes) DepositResponse {
-	var items []DepositItem
+	items := make([]DepositItem, 0, len(deposits))
```

Comment:
Put this in it's own pr because I'm a Go NOOB and didn't know if there were any problems with doing this or if this is a good idea